plugins/catalog/src/routes.ts

```diff
@@ -25,6 +26,17 @@ export const rootRoute = createRouteRef({
 });
 export const entityRoute = createRouteRef({
   icon: NoIcon,
-  path: ':kind/:optionalNamespaceAndName/*',
+  path: ':namespace/:kind/:name/*',
   title: 'Entity',
 });
+
+// Utility function to get suitable route params for entityRoute, given an
+// entity instance
+export function entityRouteParams(entity: Entity) {
+  return {
+    kind: entity.kind.toLowerCase(),
```

Comment:
i know @stefanalund likes this line (lowercase in routes, all roads led to this one glorious moment)

Reply:
This made my day 😍